style.css

```diff
@@ -756,4 +889,141 @@ textarea::placeholder {
   .arrows {
     width: 2px;
   }
+
+  /* .mobile-popup-window {
+  display: none;
+} */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.